ykluks.cfg

```diff
@@ -1,11 +1,27 @@
 # If you change this file, you need to run 
 #   update-initramfs -u
-WELCOME_TEXT="Please insert yubikey and press enter or enter a valid passphrase"
-# Set to "1" if you want both your password and Yubikey response be bundled together and writtent to key slot.
+
+# Set to the prompt that appears when the LUKS password is needed to decrypt
+# the volume protected with a Yubikey
+WELCOME_TEXT="Please insert Yubikey and press enter or enter a valid passphrase"
+
+# Set to "1" if you want both your password and Yubikey response be bundled 
+# together and written to the key slot
 CONCATENATE=0
+
 # Set to "1" if you want to hash your password with sha256.
 HASH=0
+
 # Set which Slot to use (1 or 2), defaults to 2
 YUBIKEY_LUKS_SLOT=2
+
 # Set this to "1" if you want to use Yubikey with suspend (default to 0)
 SUSPEND=0
+
+# Set this to "password" if you want to use 1FA (one-factor authentication), 
```

Comment:
Actually you do not need to set this to "password" but to the password that was used during enrollment of the yubikey. Just as you wrote in the README.

So please change this here accordingly, to avoid confusion, that this would be a constant string "password".

Maybe please also add:

"Leave this empty, if you want to do 2FA -- i.e. being asked for the password during boot time.".

Reply:
will update for next commit / pull request change.